packages/gatsby/src/services/calculate-dirty-queries.ts

```diff
@@ -0,0 +1,22 @@
+import { calcInitialDirtyQueryIds, groupQueryIds } from "../query"
+import { IBuildContext } from "../state-machines/develop"
+
+export interface IGroupedQueryIds {
+  pageQueryIds: string[]
+  staticQueryIds: string[]
+}
+
+export async function calculateDirtyQueries({
+  store,
+  filesDirty,
+}: IBuildContext): Promise<{
+  queryIds: IGroupedQueryIds
+}> {
+  const state = store?.getState()
+  if (filesDirty) {
+    // Do stuff
+  }
```

Comment:
Will we actually get list of dirty files in `develop`?

Reply:
No, this is a flag that's flipped when an invalid hook is called. We're not coordinating webpack at the moment, so it's unused.